frontend/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+import { ThemeProvider } from 'styled-components';
+import { useState, useRef } from 'react';
+import useOnClickOutside from 'hooks/useOnClickOutside';
```

Comment:
This is your local dependency, so it should be placed one line below

Reply:
True! Fixed.